enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -180,6 +180,20 @@ This controller shapes the static pod manifest itself.  It has some unique featu
     It waits until it is able to know that it is in the member list before trying to start.
     When it does this, it is able to determine the member list to use to launch.
 
+### Render Command
+Populates the static pod manifest used on the bootstrap node and other resource dependencies during the cluster bootstrap.
+
+#### File Reads
+1.) cluster-network-file - to determine the `ClusterCIDR` and `ServiceCIDR`. These values are used to conclude if the cluster is
+    single stack.
```

Comment:
list choices

1. single-stack ipv4
2. single-stack ipv6
3. dual-stack ipv4
4. dual-stack ipv6

indicate which work today and which don't and how you make the choice

Reply:
"dual-stack" means both ipv4 and ipv6 so there are only 3 choices.

I guess in theory "dual stack with ipv4 cidrs listed first" and "dual stack with ipv6 cidrs listed first" might behave slightly differently at the cluster level but I don't think it matters for etcd.